TYPO3-Solr/ext-solr
# [TASK] Provide N-Gram Filter for strings

# What this pr does

Provides a new field type and dynamic fields for strings
with enabled Edge-N-Gram filter.

Now the following fields can be used:
- *_stringEdgeNgramS
- *_stringEdgeNgramM


# How to test

Use the Solr Analysis module to check the existence and behavior of the new field type `stringEdgeNgram`


Resolves: #3035

# Note: DON'T SQUASH!
